### ISSUE TITLE
Player API doesn't work w/ "Showcases" (formerly Albums)

### ISSUE THREAD

**Issue by Corvustech** [vimeo/player.js#457]
### Expected Behavior
When I source a Showcase, events such as _progress_  can be accessed as if with any other video,
### Actual Behavior
Event handlers do not respond to the player events at all
### Steps to Reproduce
Embed a player.   Source the video from a showcase, then attempt to access the video data by displaying the output in a  label.

**Comment by luwes** [comment]
thanks for the feedback @Corvustech , do you have a test page online?

**Comment by Corvustech** [comment]
Unfortunately no, this was discovered on an environment I cannot publicize.   However, I can confirm no events from the video are caught, and the GET request for the new video is tossed into the console as a 404 (though that much may be because the Showcase is private and operating on a permitted domain).

**Comment by mandeldl** [comment]
Hi @Corvustech could you clarify what you mean by `source a Showcase`?

Are you linking to the video using
```
<iframe src="https://player.vimeo.com/showcase/12345/video/12345" />
```
as opposed to
```
<iframe src="https://player.vimeo.com/video/12345" />
```

Just trying to reproduce this on our end.  Thanks!

**Comment by Corvustech** [comment]
We attempted the latter, as well as the former (but without "video" between the Playlist and the private vid ID).

**Comment by dthomas2063** [comment]
I need to track when a video is playing on our site to prevent inactivity logout.  Unfortunately all of our embeds are showcases.  Is there a work around if the only thing I need to know is when any video is playing?

**Comment by luwes** [comment]
thanks for the feedback, we've enabled the player API for the showcase embeds today.
please let us know if you see any issues.

**Comment by mattgilbertnet** [comment]
> thanks for the feedback, we've enabled the player API for the showcase embeds today.
> please let us know if you see any issues.

Is there any documentation for this?

**Comment by luwes** [comment]
@mattgilbertnet sorry, currently no extra docs on this feature is available but it should mostly work in the same way as a regular player embed.

**Comment by mattgilbertnet** [comment]
> @mattgilbertnet sorry, currently no extra docs on this feature is available but it should mostly work in the same way as a regular player embed. please let us know if you have any questions.

I guess I was expecting some player functions like goToShowcaseItem or goToVideo or something.

If I wanted to start an embed at a certain video, is there a URL arg for that? Something like https://player.vimeo.com/showcase/12345/video/12345 ?

**Comment by ngx-developer** [comment]
@luwes  Can you please share an example on how to embed showcase

I have tried replacing the id with the album id (similar to regular video embed)

const player = new Player('handstick', {
    id: 12345, <--- album/showcase id goes here
    width: 640
});

player.on('play', function() {
    console.log('played the video!');
});

**Comment by luwes** [comment]
@ngx-developer this only works with an iframe embed, no need to add an id in the Player constructor.
https://github.com/vimeo/player.js#pre-existing-player

**Comment by luwes** [comment]
voila https://codesandbox.io/s/vimeo-showcase-player-api-zhy0l

**Comment by ngx-developer** [comment]
@luwes 
Thanks for sharing the code .
I was able to make it work partially. I have to autoplay the showcase and in loop.

<iframe src="https://vimeo.com/showcase/2038075/embed?background=1&autoPlay" [width]="videoWidth"
        [height]="videoHeight" style="position: fixed"
       frameborder="0" allowfullscreen allow="autoplay; encrypted-media"></iframe>

It freezes after the first video is played.
I tried to play the video incase is gets paused for some reason.

 player.on('pause', function() {
                        console.log('played the video!');
                        player.play();
                    });

It seems to work sometimes and also keeps throwing me an error saying
"PlayInterrupted: The play() request was interrupted by a call to pause()
PlayInterrupted: The play() request was interrupted by a call to pause()"
